snapcraft/plugins/nodejs.py

```diff
@@ -25,40 +25,35 @@
 
 Additionally, this plugin uses the following plugin-specific keywords:
 
-    - node-packages:
-      (list)
-      A list of dependencies to fetch using npm.
-    - node-engine:
+    - nodejs-version:
       (string)
       The version of nodejs you want the snap to run on.
-    - npm-run:
-      (list)
-      A list of targets to `npm run`.
-      These targets will be run in order, after `npm install`
-    - npm-flags:
-      (list)
-      A list of flags for npm.
-    - node-package-manager
-      (string; default: npm)
+      This includes npm, as would be downloaded from https://nodejs.org
+      Defaults to the current LTS release.
+
+    - nodejs-package-manager
+      (string; default: yarn)
       The language package manager to use to drive installation
-      of node packages. Can be either `npm` (default) or `yarn`.
+      of node packages. Can be either `npm` or `yarn` (default).
+
+    - nodejs-yarn-version:
```

Comment:
Judging from the URLs available below, npm supports installing specific versions as well. Do we want to expose that, here? Or is that included in the `nodejs-version`? (I'm not a node dev, if you couldn't tell).

Reply:
the nodejs bundle we download has node and npm